pkg/services/featuremgmt/toggles_gen_test.go

```diff
@@ -45,6 +45,9 @@ func TestFeatureToggleFiles(t *testing.T) {
 			if flag.Name != strings.TrimSpace(flag.Name) {
 				t.Errorf("flag Name should not start/end with spaces.  See: %s", flag.Name)
 			}
+			if flag.Stage == FeatureStageGeneralAvailability && flag.AllowSelfServe == nil {
```

Comment:
can we add the opposite check too?

and maybe another one to say if it requires restart... then can not allow self serve?  the explosion of options here makes me pretty nervous 😬

Reply:
Also, `RequiresRestart` is unused right now. All toggles require a restart, which we are handling with the webhook.